src/libraries/System.Diagnostics.Process/ref/System.Diagnostics.Process.cs

```diff
@@ -69,6 +69,8 @@ public partial class Process : System.ComponentModel.Component, System.IDisposab
         [System.ObsoleteAttribute("Process.PrivateMemorySize has been deprecated because the type of the property can't represent all valid results. Use System.Diagnostics.Process.PrivateMemorySize64 instead.")]
         public int PrivateMemorySize { get { throw null; } }
         public long PrivateMemorySize64 { get { throw null; } }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("ios")]
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("tvos")]
```

Comment:
How come all the other APIs (including `ProcessThread.PrivilegedProcessorTime`) has `SupportedOSPlatformAttribute("maccatalyst")`, but this one doesn't?

Reply:
/cc @buyaa-n @dotnet/area-system-diagnostics-process

Reply:
I'm just making them consistent with https://github.com/dotnet/runtime/blob/6a889d234267a4c96ed21d0e1660dce787d78a38/src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.iOS.cs#L32-L38

I opened https://github.com/dotnet/runtime/issues/68729